scripts/Jenkinsfile

```diff
@@ -23,10 +23,15 @@ void buildPelux(String variant, String imageName) {
         sh "rm -rf pelux-manifests/"
         sh "git clone https://github.com/Pelagicore/pelux-manifests.git -b ${branchName}"
         println("Using the specified branch for build: ${branchName}")
+        sh "cp -R ${peluxDir} pelux-manifests/${peluxDir}"
```

Comment:
Hum, is this copy needed? Could you not bind mount the workspace dir instead of $WORKSPACE/pelux-manifests? If I remember correctly, copying is done in yocto.groovy as well.

Reply:
It was moved from yocto.groovy here, as all other operations will be performed inside container

Reply:
OK, I see now that you have changed it from a `cp` in `yocto.groovy` to a `mv` in `yocto2.groovy`. (I did not have time to look at https://github.com/Pelagicore/pelux-manifests/pull/371 yesterday before it was merged.)

Reply:
I guess we can tweak things like this later if we want to. Resolving.

Reply:
I apologize